syz-manager/manager.go

```diff
@@ -322,11 +349,11 @@ func (mgr *Manager) vmLoop() {
 	for i := range instances {
 		instances[i] = vmCount - i - 1
 	}
+	mgr.hubReproQueue = make(chan *Crash, 1)
```

Comment:
It's unlikely to happen, but strictly saying this races with uses of hubReproQueue in hubSync. hubSync can potentially send to a nil queue which will crash manager.